ecs-cli/modules/commands/local/local_command.go

```diff
@@ -38,6 +38,7 @@ func LocalCommand() cli.Command {
 		Name:   "local",
 		Usage:  "Run your ECS tasks locally.",
 		Before: app.BeforeApp,
+		Flags:  flags.AppendFlags(flags.OptECSProfileFlag(), flags.OptAWSProfileFlag(), flags.OptRegionFlag()),
```

Comment:
As discussed, I think the cluster config flag should be here as well, but I understand its a bit weird, and won't block this review on it

Reply:
I don't see how adding cluster config here makes sense, even from a consistency standpoint?